tests/behaviour/util.rs

```diff
@@ -20,7 +20,40 @@
  */
 
 use cucumber::gherkin::Step;
+use std::collections::HashMap;
+use typedb_client::concept::*;
 
 pub fn iter_table(step: &Step) -> impl Iterator<Item = &str> {
     step.table().unwrap().rows.iter().flatten().map(String::as_str)
 }
+
+pub fn iter_map_table(step: &Step) -> impl Iterator<Item = HashMap<&String, &String>> {
+    let mut all_rows = step.table().unwrap().rows.iter();
+    let keys = all_rows.next().unwrap();
+    all_rows.map(|row| HashMap::from_iter(keys.iter().zip(row.iter()).collect::<Vec<_>>().into_iter()))
+}
+
+pub fn concept_as_string(concept: &Concept) -> String {
+    match concept {
+        Concept::Type(type_) => match type_ {
+            Type::Thing(thing) => match thing {
+                ThingType::Relation(RelationType { label }) => String::from(format!("label:{label}")),
+                ThingType::Entity(EntityType { label }) => String::from(format!("label:{label}")),
+                ThingType::Attribute(attr) => match attr {
+                    AttributeType::String(StringAttributeType { label }) => String::from(format!("label:{label}")),
+                    AttributeType::Long(LongAttributeType { label }) => String::from(format!("label:{label}")),
+                    AttributeType::Double(DoubleAttributeType { label }) => String::from(format!("label:{label}")),
+                    _ => String::from(""),
```

Comment:
Should either be `unreachable!()` or `todo!()`, depending on the reason for `_`

Reply:
Fixed